catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -311,8 +346,13 @@ private String safeGetBody(Request req) {
         return ImmutableMap.of(RESP_MSG, "Successfully updated");
       }
     }
-    catalogFramework.create(new CreateRequestImpl(metacard));
-    return ImmutableMap.of(RESP_MSG, "Successfully created");
+    Metacard createdMetacard =
+        catalogFramework.create(new CreateRequestImpl(metacard)).getCreatedMetacards().get(0);
+    return ImmutableMap.of(
+        "id",
+        createdMetacard.getAttribute("id").getValue(),
```

Comment:
✏️ `Core.ID`

Reply:
Oh yeah, thanks for the reminder! I meant to go back and do that but forgot